azurelinuxagent/pa/deprovision/default.py

```diff
@@ -35,10 +35,10 @@
 
 
 def read_input(message):
-    if sys.version_info[0] >= 3:
+    if sys.version_info >= (3,):
         return input(message)
     else:
-        return raw_input(message)
+        return raw_input(message)  # noqa
```

Comment:
can we add a comment about what is being ignored?

Reply:
I added one locally.  flake8 doesn't see the branch and complains about NameError on raw_input (F821).  This is normal for system version specific code with flake8.